prepare-assets.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+"""
+Prepare assets for deployment
+- --default: ensure ./public/pro is absent.
+- --pro: merge ./public/pro into ./public (overwriting files), keep originals unless explicitly replaced,
+         and copy index-pro.html to index.html.
+"""
+from __future__ import annotations
+import argparse
+import shutil
+from pathlib import Path
+
+ROOT = Path(__file__).resolve().parent
+
+PUBLIC = ROOT / "public"
+PRO = PUBLIC / "pro"
+
+def merge_dir(src: Path, dst: Path):
+    if not src.exists():
+        return
+    for p in src.rglob("*"):
+        rel = p.relative_to(src)
+        target = dst / rel
+        target.parent.mkdir(parents=True, exist_ok=True)
+        shutil.copy2(p, target)
+
+def main():
```

Comment:
This is a reaslly bad idea. How do you make sure those two index.html files stay in sync?